google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITStorageSnippets.java

```diff
@@ -435,6 +437,16 @@ public void testBlobDownload() throws Exception {
     assertArrayEquals(BLOB_BYTE_CONTENT, readBytes);
   }
 
+  @Test
+  public void testGetBlobMetadata() {
```

Comment:
These tests don't really test the snippet, they test the same thing the snippet is running. Ideally you should run the snippet and check the output for indicators that everything worked well.

Reply:
I wanted to keep this test simple, but I'll add that in.